.github/workflows/e2e.yml

```diff
@@ -9,17 +9,20 @@ jobs:
     strategy:
       matrix:
         include:
-        - docker_base_image: 'ubuntu:20.04'
-          gift_ppa_track: 'staging'
-        - docker_base_image: 'ubuntu:20.04'
-          gift_ppa_track: 'stable'
+          - docker_base_image: "ubuntu:20.04"
+            gift_ppa_track: "staging"
+          - docker_base_image: "ubuntu:20.04"
+            gift_ppa_track: "stable"
     steps:
       - uses: actions/checkout@v2
       - name: Pull docker images for cache
         run: |
           docker pull ${{ matrix.docker_base_image }}
+          docker pull us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-api-server-dev:latest
           docker pull us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-server-dev:latest
           docker pull us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-worker-dev:latest
+      - name: Build Turbinia server Docker image
```

Comment:
Maybe s/server/API server/ here

Reply:
looks like that particular issue is fixed now as e2e stable tests pass (though staging is failing, unsure why at this point).